jsapp/js/components/common/bigSpinner.component.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import styles from './bigSpinner.module.scss';
+
+/** Displays an animated blue spinner - for white backgrounds only. */
+export default function BigSpinner() {
+  return (
+    <span className={styles.bigSpinner}/>
+  );
+}
```

Comment:
Does this need to be a separate component from `<LoadingSpinner/>`? The only real difference I see is the style/size of the spinner element/icon, and it's missing a label (visible or just ARIA-accessible), which `<LoadingSpinner/>` has. It might be worth considering adding this as a size option for `<LoadingSpinner/>` - or you could just give it some sort of accessible label.

Reply:
Didn't think about incorporating it into LoadingSpinner. Good idea. I ended up improving it in greater deal than just adding big version to it :)